src/modules/entity/reducer.ts

```diff
@@ -56,7 +58,33 @@ export function entityReducer(state: EntityState = INITIAL_STATE, action: Entity
         error: null
       }
     }
-    case FETCH_ENTITIES_BY_POINTERS_SUCCESS:
+    case FETCH_ENTITIES_BY_POINTERS_SUCCESS: {
+      const { entities, pointers } = action.payload
+
+      const entityPointersSet = new Set(entities.flatMap(entity => entity.pointers))
+
+      const newMissingEntities = { ...state.missingEntities }
+
+      for (const pointer of pointers) {
+        if (!entityPointersSet.has(pointer)) {
+          newMissingEntities[pointer] = true
+        }
+      }
+
+      return {
+        ...state,
+        loading: loadingReducer(state.loading, action),
+        error: null,
+        data: {
+          ...state.data,
+          ...entities.reduce((obj, entity) => {
+            obj[entity.id] = entity
+            return obj
+          }, {} as EntityState['data'])
+        },
+        missingEntities: newMissingEntities
```

Comment:
What if instead of storing the missing entities, we create a new memoized selector, that by using the items and the entities, returns the newMissingEntities map?